server/packetio.go

```diff
@@ -59,16 +62,26 @@ type packetIO struct {
 	// maxAllowedPacket is the maximum size of one packet in readPacket.
 	maxAllowedPacket uint64
 	// accumulatedLength count the length of totally received 'payload' in readPacket.
-	accumulatedLength uint64
+	accumulatedLength    uint64
+	compressionAlgorithm int
+	compressedSequence   uint8
+	zstdLevel            zstd.EncoderLevel
```

Comment:
will it be changed to other values than 3?

Reply:
Fixed in 16a3079114b4161b6e6db50c243bfadc51bcf51c

Also note that there is a translation here:

```go
	for i := 1; i <= 22; i++ {
		fmt.Printf("%d: %#v\n", i, zstd.EncoderLevelFromZstd(i))
	}
```
```
1: 1
2: 1
3: 2
4: 2
5: 2
6: 3
7: 3
8: 3
9: 3
10: 4
11: 4
12: 4
13: 4
14: 4
15: 4
16: 4
17: 4
18: 4
19: 4
20: 4
21: 4
22: 4
```